### ISSUE TITLE
Upgrade from old FM: Error with creating UC_Version because PK_Version already exists

### ISSUE THREAD

**Issue by markwhitfeld** [fluentmigrator/fluentmigrator#359]
I upgraded Fluent Migrator from a much older version and it had issues upgrading the VersionInfo Table.
Upon running migrations after upgrade from v1.0.0 (I think) to v1.0.6:

```
System.Exception : An error occured executing the following sql:
CREATE UNIQUE CLUSTERED INDEX [UC_Version] ON [dbo].[VersionInfo] ([Version] ASC)
The error was Cannot create more than one clustered index on table 'dbo.VersionInfo'. Drop the existing clustered index 'PK_VersionInfo' before creating another.

  ----> System.Data.SqlClient.SqlException : Cannot create more than one clustered index on table 'dbo.VersionInfo'. Drop the existing clustered index 'PK_VersionInfo' before creating another.
   at FluentMigrator.Runner.Processors.SqlServer.SqlServerProcessor.ExecuteNonQuery(String sql) in SqlServerProcessor.cs: line 186
   at FluentMigrator.Runner.MigrationRunner.AnnounceTime(String message, Action action) in MigrationRunner.cs: line 391
   at FluentMigrator.Runner.MigrationRunner.ExecuteExpressions(ICollection`1 expressions) in MigrationRunner.cs: line 381
   at FluentMigrator.Runner.MigrationRunner.Up(IMigration migration) in MigrationRunner.cs: line 316
   at FluentMigrator.Runner.VersionLoader.LoadVersionInfo() in VersionLoader.cs: line 129
   at FluentMigrator.Runner.MigrationRunner..ctor(Assembly assembly, IRunnerContext runnerContext, IMigrationProcessor processor) in MigrationRunner.cs: line 63

Basically my current VersionInfo table lookes like this (in SQLServer):
CREATE TABLE [dbo].[VersionInfo](
    [Version] [bigint] NOT NULL,
 CONSTRAINT [PK_VersionInfo] PRIMARY KEY CLUSTERED 
(
    [Version] ASC
)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
) ON [PRIMARY]
```

In the Version Loader code, the check to see if the unique constraint is created looks to see if the 'AppliedOn' column is created because it is in the same migration. Surely they should be seperate and have seperate checks?
